packages/metaltone.lv2/PKGBUILD

```diff
@@ -0,0 +1,45 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+_name=MetalTone
+pkgname=${_name,,}.lv2
+pkgver=0.1
+pkgrel=1
+pkgdesc='A high gain distortion with an advanced EQ section guitar pedal LV2 plugin'
+arch=(aarch64 x86_64)
+url="https://github.com/brummer10/$_name"
+license=(0BSD GPL3)
```

Comment:
namcap complains "0BSD is not a common license", but https://wiki.archlinux.org/title/PKGBUILD#license states:

> License families like BSD or MIT are, strictly speaking, not a single license and each instance requires a separate license file. In license variable refer to them using a common SPDX identifier (e.g. 'BSD-3-Clause' or 'MIT'), but then provide the corresponding file as if it was a custom license.